src/syscheckd/config.c

```diff
@@ -80,6 +80,21 @@ int Read_Syscheck_Config(const char *cfgfile)
     ReadConfig(modules, AGENTCONFIG, &syscheck, NULL);
 #endif
 
+    int it = 0;
+
+    /* Check directories options to determine whether to start the whodata thread or not */
+    if (syscheck.dir) {
```

Comment:
I think this code is redundant. When FIM starts, a series of messages are printed informing about the configuration of the directories. This check should be done in `src/syscheckd/main.c:234`.

Reply:
Done 186bb0b